etcd/src/main/scala/com/github/fit51/reactiveconfig/etcd/EtcdWatch.scala

```diff
@@ -67,6 +67,7 @@ trait Watch[F[_]] {
 
       override def onNext(elem: WatchResponse): Future[Ack] = elem match {
         case v: WatchResponse if !v.created && !v.canceled =>
+          logger.info(s"Updated etcd keys: ${v.events.flatMap(_.kv.map(_.key.toStringUtf8)).mkString(",")}")
```

Comment:
It prints information about all updated keys. Even if application doesn't have Reloadable subscribed on some key.
It will not work for typesafe config.

Maybe you should implement this inside core project:
```
@@ -23,19 +25,18 @@ import scala.concurrent.Future
 import scala.util.control.NonFatal
 import scala.util.Failure
 
-object Reloadable {
+object Reloadable extends StrictLogging {
   def apply[F[_]: TaskLike: TaskLift, A](
       initial: A,
-      ob: Observable[A]
+      ob: Observable[A],
+      mbKey: Option[String] = None
   )(implicit scheduler: Scheduler, F: MonadError[F, Throwable]): F[Reloadable[F, A]] =
     (for {
       connectableObservable <- Task.pure(ob.publish)
       canceler   = connectableObservable.connect()
       reloadable = new ReloadableImpl[F, A](initial, connectableObservable)
       fiber <- connectableObservable
-        .consumeWith(Consumer.foreach { newValue =>
-          reloadable.value = newValue
-        })
+        .consumeWith(Consumer.foreachEval(reloadable.modifyCurrentValue(mbKey, _)))
         .start
     } yield {
       reloadable.canceler = (fiber.cancel >> Task.delay(canceler.cancel())).to[F]
@@ -294,6 +314,14 @@ private class ReloadableImpl[F[_]: TaskLike: TaskLift, A](initial: A, ob: Observ
   override def stop: F[Unit] =
     canceler
 
+  private[reloadable] def modifyCurrentValue(mbKey: Option[String], value: A): F[Unit] =
+    Task
+      .delay({
+        mbKey.foreach(key => logger.info(s"Updated key $key"))
+        self.value = value
+      })
+      .to[F]
+
   override def mapK[G[_]: TaskLike: TaskLift: MonadError[?[_], Throwable]]: G[Reloadable[G, A]] =
     Reloadable[G, A](initial, ob)

```

Reply:
Yeah. This implementation is much better